### PR TITLE
Fix pledge met filter

### PR DESCRIPTION
**Pull request by iccicci**
# Context

Stake pool queries with `pledgeMet` filter exclude some pools from the result.

Found a minor bug on stake pool query sort by name: the `->` operator doesn't convert the field, but only extracts a property as json type. This is originating a minor problem: the double quotes of json representation of string was included in the sort algorithm.

# Proposed Solution

Added a `LEFT` clause to the failing `JOIN`.

Changed `->` to `->>` in the `SORT` clause of stake pools query sort by name.

# Important Changes Introduced

- added dev version of _mainnet_, _preprod_ and _preview_ docker compose;
- fixed a bug in stake pools query which was causing a division by zero;
- fixed sort by name stake pools query e2e tests to be case-insensitive;

### CODE REVIEW

File: packages/e2e/docker-compose-dev.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
-version: "3.9"
-
 services:
   http-server:
     volumes:
-      - ..:/app/packages
+      - ../..:/app
```
Comment (rhyslbw) [inline packages/e2e/docker-compose-dev.yml:4]: These changes are in the wrong commit

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looks good so far @iccicci

File: packages/e2e/docker-compose-dev.yml (hunk 1 above)
Comment (iccicci) [reply packages/e2e/docker-compose-dev.yml:6]: 18b0bc171 plus 8643ce86a should solve this

**Review by rhyslbw — APPROVED** [review]
LGTM!

**Review by IvayloAndonov — APPROVED** [review]
LGTM as well, good new knowledge for the `->>` operator :)

**Review by IvayloAndonov — APPROVED** [review]

**Pull request merged**